library/kernel_settings.py

```diff
@@ -171,42 +171,53 @@ def wrapper_set_logger_class(_clsname):
 EXAMPLES = """
 # add the sysctl vm.max_mmap_regions, and disable spectre/meltdown security
 kernel_settings:
-    sysctl:
-      - name: vm.max_mmap_regions
-        value: 262144
-    sysfs:
-      - name: /sys/kernel/debug/x86/pti_enabled
-        value: 0
-      - name: /sys/kernel/debug/x86/retp_enabled
-        value: 0
-      - name: /sys/kernel/debug/x86/ibrs_enabled
-        value: 0
+  sysctl:
+    - name: vm.max_mmap_regions
+      value: 262144
+  sysfs:
+    - name: /sys/kernel/debug/x86/pti_enabled
+      value: 0
+    - name: /sys/kernel/debug/x86/retp_enabled
+      value: 0
+    - name: /sys/kernel/debug/x86/ibrs_enabled
+      value: 0
 
 # replace the existing sysctl section with the specified section
 # delete the /sys/kernel/debug/x86/retp_enabled setting
 # completely remove the vm section
 # add the bootloader cmdline arguments spectre_v2=off nopti
 # remove the bootloader cmdline arguments quiet splash
 kernel_settings:
-    sysctl:
-      - previous: replaced
-      - name: vm.max_mmap_regions
-        value: 262144
-    sysfs:
-      - name: /sys/kernel/debug/x86/retp_enabled
-        state: absent
-    vm:
-      state: empty
-    bootloader:
-      - name: cmdline
-        value:
-          - name: spectre_v2
-            value: off
-          - name: nopti
-          - name: quiet
-            state: absent
-          - name: splash
-            state: absent
+  sysctl:
+    - previous: replaced
+    - name: vm.max_mmap_regions
+      value: 262144
+  sysfs:
+    - name: /sys/kernel/debug/x86/retp_enabled
+      state: absent
+  vm:
+    state: empty
+  bootloader:
+    - name: cmdline
+      value:
+        - name: spectre_v2
+          value: off
+        - name: nopti
+        - name: quiet
+          state: absent
+        - name: splash
+          state: absent
+"""
+
+RETURN = """
+message:
+    A short text message to say what action this module performed.
+new_profile:
+    This is the tuned profile in dict format, after the changes, if any,
+    have been applied.
+reboot_needed:
```

Comment:
Please name it `reboot_required` for consistency with prior art here: https://docs.ansible.com/ansible/latest/modules/selinux_module.html#return-reboot_required